doc/lua_api.md

```diff
@@ -613,13 +613,13 @@ Creates an inventorycube with `grass.png`, `dirt.png^grass_side.png` and
 * `<y>`: y position
 * `<color>`: a `ColorString`.
 
```

Comment:
if you're doing this anyway please document that [combine x or y can be negative